### ISSUE TITLE
Feat: Windows Explorer Freezing Due to Unreachable Network File Associations

### ISSUE THREAD

**Issue by SandiyosDev** [LeDragoX/Win-Debloat-Tools#111]
### Description

Hey, I wanted to bring up a performance issue in Windows related to file associations on unreachable network disks. This problem occurs when a user tries to interact with a file type in Windows Explorer that has no associated default program. If the selected program is located on a network disk that becomes unreachable, Windows Explorer experiences significant freezing and delays, affecting not only file opening operations but also basic interactions like highlighting files or accessing the root directory of a local disk.

I believe this issue could be affecting other users in networked environments as well. So, I am proposing a solution that could be incorporated into one of the existing PowerShell scripts in the Win-Debloat-Tools repository.

The proposed tweak would involve the script systematically checking all file association records. If an association links to a program on an unreachable network disk, the script would remove that file association record. This should help alleviate the freezing issues associated with unreachable file association targets.

I initially tried to create a pull request for this proposal, but received a validation error stating that I must be a collaborator. Hence, I'm creating this issue to open a discussion about the problem and the proposed solution. Let me know if I could be given the go-ahead, If the proposal seems beneficial, I am more than willing to work on implementing this new feature.

### Reason

N/A

**Comment by LeDragoX** [comment]
I know i've been disappeared from this website, but this detail you've said you got a validation error and isn't able to collaborate was unknown to me, i want people to be able to interact with this repository, even if i'm studying or doing other things not related to programming.

I don't known why this happened, and will need help to remove that "validation restriction". I'm commenting before getting busy, so i may take some time to respond again.

**Comment by SGprooo** [comment]
The exact error that I'm getting is "![Pull request creation failed. Validation failed: must be a collaborator](https://github.com/LeDragoX/Win-Debloat-Tools/assets/25407129/296b35fe-c9f2-4844-8974-91a1c4db3463)", simply looking it up, it may occur as a preventative measure against potential spam or misuse, or perhaps there are some other permission errors blocking the pull from being created.
However, adding someone as a collaborator does involve risks, as Collaborators can push changes directly to the repository, though I'm not sure what the exact issue is, I'd suggest that you check the repository permission settings or reach out to GitHub directly, I'll then work on adding this functionality soon after.

**Comment by LeDragoX** [comment]
This was occurring by the default setting GitHub added.
![image](https://github.com/LeDragoX/Win-Debloat-Tools/assets/27643047/53fd43ac-bea9-4fe8-a91d-e01bd47f1ac9)
I disabled branch protection, but I still enforce people should send commits to the `develop` branch, mostly for control, to not let things run out of control, I wouldn't want a lot of issues happening cause the `main` branch got compromised.

I got this PR recently #121, so this shouldn't be causing any trouble.
